### ISSUE TITLE
YoutubeAudioSourceManager: IllegalStateException: Get only works on a map

### ISSUE THREAD

**Issue by robinfriedli** [sedmelluq/lavaplayer#201]
Hi!

I am hosting a music bot using JDA and lavaplayer and since updating to the latest version I'm getting a lot of those errors (well 13 back to back with a 2-hour distance between first and last) I don't think I've seen in previous versions. What's weird is that all of those YouTube videos that caused these errors seem to be unavailable now, so, unfortunately, I couldn't reproduce the issue. The line that throws this exception has apparently changed in the last update so I figured that maybe something broke with certain videos. The YouTube videos are all loaded via the YouTube API before passing the URL to lavaplayer, so it's unlikely that the videos were already unavailable when these errors were thrown. Going through the command history I noticed that it might have something do to with YouTube playlists being played that contain private videos, though I think in past lavaplayer version I used (1.3.17) the error for private videos was different and more coherent.

```
[ERROR] 2019-07-31 15:22:58.384 [lava-daemon-pool-info-loader-2-thread-1] UrlAudioLoader - lavaplayer threw an exception while loading track https://www.youtube.com/watch?v=_FrD_9mHuP0
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Loading information for a YouTube track failed.
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:226) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadFromUrlWithVideoId(YoutubeAudioSourceManager.java:316) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadFromMainDomain(YoutubeAudioSourceManager.java:253) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadNonSearch(YoutubeAudioSourceManager.java:323) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:189) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:128) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218) ~[lavaplayer-1.3.19.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.IllegalStateException: Get only works on a map
	at com.sedmelluq.discord.lavaplayer.tools.JsonBrowser.get(JsonBrowser.java:62) ~[lavaplayer-1.3.19.jar:?]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:221) ~[lavaplayer-1.3.19.jar:?]
	... 12 more
```

**Comment by robinfriedli** [comment]
I tested a playlist that contains private videos which apparently does seem to be the cause of this issue.

**Comment by duncte123** [comment]
Got the same issue on 1.3.19
https://sentry.io/share/issue/a27b97ab85b747f1bda2a573ff6a2ddf/

**Comment by aikaterna** [comment]
Disclaimer: I am a Lavalink user, although I am using Lavaplayer 1.3.19. My node is hosted in the US-East region.

https://www.youtube.com/watch?v=V-MixcXotj8 
https://www.youtube.com/watch?v=ic2d-1xVkkI
Video unavailable - generic error

https://www.youtube.com/watch?v=cU8HrO7XuiE
Video unavailable - copyright claim
Should return FriendlyException instead: This video is no longer available due to a copyright claim by SME.

https://www.youtube.com/watch?v=AQBh9soLSkI
Video unavailable - copyright claim
Should return FriendlyException instead: This video is no longer available due to a copyright claim by Studio Chizu.

https://www.youtube.com/watch?v=_hI0qMtdfng
No Video unavailable header, just what should be returned in the FriendlyException: This video has been removed for violating YouTube’s policy on hate speech. Learn more about combating hate speech in your country.


```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Loading information for a YouTube track failed.
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:226) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadFromUrlWithVideoId(YoutubeAudioSourceManager.java:316) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadFromMainDomain(YoutubeAudioSourceManager.java:253) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadNonSearch(YoutubeAudioSourceManager.java:323) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:189) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:128) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218) ~[lavaplayer-1.3.19.jar!/:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_212]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_212]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_212]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_212]
Caused by: java.lang.IllegalStateException: Get only works on a map
	at com.sedmelluq.discord.lavaplayer.tools.JsonBrowser.get(JsonBrowser.java:62) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:221) ~[lavaplayer-1.3.19.jar!/:na]
	... 12 common frames omitted

```

**Comment by Devoxin** [comment]
This is caused by unavailable videos. Lavaplayer should actually return before it reaches the part that calls `.get`, but isn't because YouTube moved some properties around. This issue should be fixed by #205
